### PR TITLE
[#1198] workaround for keep local history change in platform

### PR DESCRIPTION
**Pull request by cdietrich**
[#1198] workaround for keep local history change in platform
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: org.eclipse.xtend.ide.tests/src/org/eclipse/xtend/ide/tests/builder/KeepLocalHistoryTest.xtend [hunk 1]
```diff
@@ -162,6 +165,21 @@ class KeepLocalHistoryTest extends AbstractXtendUITestCase {
 
 	def setKeepLocalHistory(boolean keepLocalHistory) {
 		setValue(OUTPUT_KEEP_LOCAL_HISTORY, keepLocalHistory)
```
Comment (szarnekow) [inline org.eclipse.xtend.ide.tests/src/org/eclipse/xtend/ide/tests/builder/KeepLocalHistoryTest.xtend:167]: Should we add a warning to the preference page that keepLocalHistory and markAsDerived contradict each other in newer eclipses?

Comment (cdietrich) [reply org.eclipse.xtend.ide.tests/src/org/eclipse/xtend/ide/tests/builder/KeepLocalHistoryTest.xtend:167]: fix needs to be done in xtext-eclipse anyway

Comment (cdietrich) [reply org.eclipse.xtend.ide.tests/src/org/eclipse/xtend/ide/tests/builder/KeepLocalHistoryTest.xtend:167]: https://github.com/eclipse/xtext-eclipse/pull/1680/files

**Comment by cdietrich** [comment]
see also https://github.com/eclipse/xtext-eclipse/pull/1680/files

**Review by szarnekow — APPROVED** [review]

**Pull request merged**